dts/arm/nxp/nxp_rt10xx.dtsi

```diff
@@ -991,6 +991,34 @@
 			nxp,rx-dma-channel = <6>;
 			status = "disabled";
 		};
+
```

Comment:
I'd change the title of this commit to "dts: arm: add qdec nodes and update soc". The commit message also needs to be updated, as the `qdec` Kconfig is no longer added to the RT1050

Reply:
Title and description updated